nwx_build_environment/add_exachem.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nwx_buildenv:latest
+
+ARG VERSION=main
+
+# Install exachem (add dependencies)
+RUN cd /tmp \
+    && git clone https://github.com/ExaChem/exachem.git \
+    && cd exachem \
+    && git checkout ${VERSION} \
+    && bash /scripts/tamm_command.sh \
```

Comment:
Is this a copy/paste error? Or does the same script work for exachem too?

Reply:
same script will work